lib/index.js

```diff
@@ -1,210 +1,143 @@
-
+var _ = require('lodash');
```

Comment:
Rather than including the whole of lodash, it would be good to include the packages needed.
e.g.
```js
var sortBy = require('lodash.sortby');
```
and amend line 87 to match the variable name.